filebeat/tests/system/test_modules.py

```diff
@@ -150,7 +150,10 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
                              bufsize=0).wait()
 
         # List of errors to check in filebeat output logs
-        errors = ["Error loading pipeline for fileset"]
+        errors = [
+            "Error loading pipeline for fileset",
```

Comment:
I think this is from a Go error which means the first letter will be lowercase. Perhaps just drop the `E` so that it matches either case.